tribe-ext-link-events-to-website-url/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar: Link Event Titles to Event Website URLs
```

Comment:
You forgot to add "Extension" after The Events Calendar. Also, could we shorten this a bit? Titles don't wrap in the WP admin screen list. Thus the title column can end up taking up the entire width of small screens if you're not careful, making the plugins page almost unusable.

Maybe "Link Events to Website URLs"? Then you can elaborate in the description as you've done.